client/src/containers/CampaignAdmin/CampaignAdmin.js

```diff
@@ -377,14 +381,20 @@ CampaignAdmin.propTypes = {
   view: React.PropTypes.string,
 };
 
-function mapStateToProps(state) {
+function mapStateToProps(state, ownProps) {
+  const sectionConfig = state.config.sections.find((section) => section.name === sectionConfigKey);
+  const formID = `${sectionConfig.form.campaignEditForm.schemaUrl}/${ownProps.params.id}`;
+  const selector = formValueSelector(formID);
+  const title = selector(state, 'Name');
```

Comment:
missing `if (this.props.params.id <= 0) {` check, which the `renderDetailEditView()` method has, could lead to side-effects later on